scene/gui/popup_menu.cpp

```diff
@@ -330,6 +330,7 @@ int PopupMenu::_get_mouse_over(const Point2 &p_over) const {
 	item_clickable_area.position.x += theme_cache.panel_style->get_margin(SIDE_LEFT);
 	item_clickable_area.position.y += theme_cache.panel_style->get_margin(SIDE_TOP);
 	item_clickable_area.position *= win_scale;
+	item_clickable_area.size.width -= theme_cache.panel_style->get_margin(SIDE_LEFT) + theme_cache.panel_style->get_margin(SIDE_RIGHT);
```

Comment:
```suggestion
	item_clickable_area.size.x -= theme_cache.panel_style->get_margin(SIDE_LEFT) + theme_cache.panel_style->get_margin(SIDE_RIGHT);
```
More consistent with the code around.

Reply:
```
		item_clickable_area.size.width -= scroll_width;
		}
		item_clickable_area.size.width -= scroll_width;
```

line 325 and 327 above uses width

Reply:
I was more looking at the line below that uses `y`. Looks like it's not consistent.